RELEASE_NOTES.rst

```diff
@@ -175,8 +175,7 @@ are added for existing bears.
 
 New bears:
 -  ``AutoPrefixBear`` (Add vendor prefixes automatically in CSS)
--  ``ClangComplexityBear`` (Calculates cyclomatic complexity of each function
```

Comment:
rstcheck revealed difference in indents so I made sure they were uniform.Nothing else.